pkg/hwevent/event_ce_test.go

```diff
@@ -133,7 +154,6 @@ func TestEvent_GetCloudNativeEvents(t *testing.T) {
 				return
```

Comment:
won't this return break the testcase loop ?

Reply:
removed